Userland/ls.cpp

```diff
@@ -300,7 +305,6 @@ bool print_filesystem_object(const String& path, const String& name, const struc
         printf("  %4u,%4u ", major(st.st_rdev), minor(st.st_rdev));
     } else {
         if (flag_human_readable) {
-            ASSERT(st.st_size > 0);
```

Comment:
This makes no sense, why was it here at all?
```sh
ls > foo -alh
```
would trip this, why do we care?

Reply:
We don't care. No idea why that was there :)